cves/CVE-2011-3192.yml

```diff
@@ -52,14 +56,10 @@ fixes:
      note:
    - commit: 58e50d77a98357b5cef0eb7b8694a165631ab87a
      note:
```

Comment:
Label as back patch of above fix for CVE-2011-3192

Reply:
Done